### ISSUE TITLE
The password generation for WebLogic on Docker image boot is not 100% reliable

### ISSUE THREAD

**Issue by craigbarrau** [oracle/docker-images#371]
The current WebLogic password generation that runs on boot is relying on the first 8 characters of an `md5sum` so there is no guarantee that this will always meet the WebLogic password requirements.

Below is the output of one such example where it failed:
```
docker run --rm -ti -p 7001:7001 container-registry.oracle.com/middleware/weblogic:12.2.1.2 

    Oracle WebLogic Server Auto Generated Admin password: bdfdfaad


Initializing WebLogic Scripting Tool (WLST) ...

Welcome to WebLogic Server Administration Scripting Shell

Type help() for help on available commands

domain_name : [base_domain]
admin_port  : [7001]
domain_path : [/u01/oracle/user_projects/domains/base_domain]
production_mode : [dev]
Problem invoking WLST - Traceback (innermost last):
  File "/u01/oracle/create-wls-domain.py", line 41, in ?
60455: The password must be at least 8 alphanumeric characters with at least one number or special character.
60455: Correct the password.
	at com.oracle.cie.domain.script.jython.WLSTSecurityPrincipal.set(WLSTSecurityPrincipal.java:73)
	at com.oracle.cie.domain.script.jython.WLSTSecurityUser.setPassword(WLSTSecurityUser.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
```

**Comment by mriccell** [comment]
@craigbarrau I created a push request #384 to correct this problem. After change has been merged please test it by creating the 12.2.1.2 image.

**Comment by brunoborges** [comment]
Let's close once PR #384 is merged
